Makefile

```diff
@@ -120,6 +120,14 @@ coverage: ## show the coverage report
 	# coverage report
 	echo no-op coverage report
 
+
 .PHONY: clean
 clean: ## clean up the environment by deleting the .venv, dist, eggs, mypy caches, coverage info, etc
-	rm -rf .venv $(DEPS) dist *.egg-info .mypy_cache build .pytest_cache .coverage runinfo $(WORKQUEUE_INSTALL)
+	parsl+flux
```

Comment:
what is this parsl+flux line for?

Reply:
Hi @benclifford this is a mistake, I'll proceed with removing it.